Packs/Whois/Integrations/Whois/Whois.yml

```diff
@@ -394,7 +313,7 @@ script:
     - contextPath: Domain.Whois.QueryStatus
       description: The result of the command ("Success" or "Failed").
       type: string
-  dockerimage: demisto/ippysocks
+  dockerimage: demisto/ippysocks:latest
```

Comment:
create a new version in `dockerfiles` repository and use the updated version here.

Reply:
This change was removed - it is unnecessary in this case.

Reply:
will do